Parser/Updating/Parser.ts

```diff
@@ -67,82 +66,91 @@ const HelpWorker = ({ print }) => {
 };
 
 const BorrowGcdWorker = createWorker(async ({ lock, loading, print }, Args, [userAddress]) => {
-  try {
-    lock(true);
-    loading(true);
-    if (!(await isCurrentChain(network))) {
-      throw new Error(`Wrong network, switch on ${chain.chainName}, please.`);
-    }
-    const tmpARGS = Args.split(' ');
+  const tmpARGS = Args.split(' ');
     if (tmpARGS.length < 7) {
       throw new Error('Invalid input');
     }
 
-    const Amount = tmpARGS[2];
-    const TokenName = tmpARGS[3];
-    const percentRisk = +parseInt(tmpARGS[5]) / 100;
+  const Amount = tmpARGS[2];
+  const TokenName = tmpARGS[3];
+  const percentRisk = +parseInt(tmpARGS[5]) / 100;
```

Comment:
We can get rid of either `+` or `parseInt()`. The only difference is that `parseInt('1234abc') -> 1234` while +`'1234abc' -> NaN` (so `+` is essentially identical to calling `Number()` on an argument)

Reply:
Fixed